src/js/elements/location_info_box.js

```diff
@@ -24,11 +27,14 @@ export class LocationInfoBox extends Observable {
     }
 
     updateHighlights(highlights) {
+        const self = this;
```

Comment:
since we only use this once for the config it's better to assign the config here and not add self to it.

Reply:
so shouldn't I assign it to this.config in contructor function?

Reply:
you don't need to use `self` or anything here. 
just use `let formattingConfig = this.formattingConfig` since only `this` will be overwritten with the local context. as long as we don't reuse `this` we don't need to use `self` here.